client/src/components/App.jsx

```diff
@@ -4,24 +4,30 @@ import Header from './Header';
 import Overview from './Overview';
 import Review from './Review';
 import queryString from 'query-string';
+import styled from 'styled-components';
+
+const ReviewsHolder = styled.div`
+  font-family: 'Montserrat', sans-serif;
+`;
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       listing: 1,
       reviews: [{
-        body: '',
-        overall: '',
-        firstname: '',
-        imageurl: ''
+        body: 'Ad error quia quos ipsa ut. Ab quisquam est nulla. Cumque optio ut voluptatem quibusdam. Fugit sequi quisquam tempore sint a est. Fugit amet adipisci. Earum quam ipsum numquam. Quo adipisci modi doloremque ex a. Doloribus aut odit. Voluptatem laborum id nobis voluptas eius quaerat unde ratione. Impedit quia accusamus fuga non accusantium ullam.',
+        overall: 4.2,
+        firstname: 'Andrew',
+        imageurl: 'https://s3.amazonaws.com/uifaces/faces/twitter/jayphen/128.jpg'
       }],
       overview: {
-        overall: 0,
+        overall: 4.2,
       },
```

Comment:
I believe you put in sample data here for your page to correctly render the first time as your get request is slower than your render, so you're getting errors in the beginning. Not sure if we'll be able to leave this sample data as hard coded, or if all data should be drawn from the database. How I got around this problem was through rendering my child components only when the state has been set to the appropriate data structure. In your case, you could do something like setting reviews to just "reviews:[]" and in your render function do {this.state.reviews[0] && <Component />}. This would render your child component only after you this.state.reviews has something in it. It does create some other problems such as having to make your tests asynchronous because otherwise your tests will run before your state is set to the database data.